.github/workflows/ci-tests.yml

```diff
@@ -70,12 +76,13 @@ jobs:
 
       - name: Install resqpy
         run: |
-          python -m pip install --upgrade pip setuptools wheel
-          pip install -e ".[tests]"
+          pip install poetry
+          pip install poetry-dynamic-versioning
+          poetry install
```

Comment:
These steps look redundant from step above. Suggest this step just installs resqpy, and previous step just installs poetry.

Reply:
The unit tests run in parallel on a separate build agent to the static analysis task, therefore poetry needs to be installed on both.

Reply:
Yes indeed you're right, I don't know what I was thinking yesterday!